iOS/Flows/Conversation/Cells/ConversationMessagesCell.swift

```diff
@@ -263,8 +263,8 @@ class ConversationMessagesCell: UICollectionViewCell, ConversationUIStateSettabl
         return self.collectionView.convert(dropZoneFrame, to: targetView)
     }
 
-    func getBottomFrontmostCell() -> MessageCell? {
-        return self.collectionLayout.getBottomFrontmostCell()
+    func getFrontmostCell() -> MessageCell? {
+        return self.collectionLayout.getFrontmostCell()
```

Comment:
There's no longer two sections so the "bottom cell" doesn't make sense any more.